FirebaseCore/Internal/FirebaseCoreInternal.h

```diff
@@ -17,12 +17,13 @@
 
 #import <FirebaseCore/FirebaseCore.h>
 
-#import "FirebaseCore/Sources/Private/FIRAppInternal.h"
-#import "FirebaseCore/Sources/Private/FIRComponent.h"
-#import "FirebaseCore/Sources/Private/FIRComponentContainer.h"
-#import "FirebaseCore/Sources/Private/FIRComponentType.h"
-#import "FirebaseCore/Sources/Private/FIRDependency.h"
-#import "FirebaseCore/Sources/Private/FIRHeartbeatInfo.h"
-#import "FirebaseCore/Sources/Private/FIRLibrary.h"
-#import "FirebaseCore/Sources/Private/FIRLogger.h"
-#import "FirebaseCore/Sources/Private/FIROptionsInternal.h"
+#import "FIRAppInternal.h"
+#import "FIRComponent.h"
+#import "FIRComponentContainer.h"
+#import "FIRComponentType.h"
+#import "FIRCoreDiagnosticsConnector.h"
+#import "FIRDependency.h"
+#import "FIRHeartbeatInfo.h"
+#import "FIRLibrary.h"
+#import "FIRLogger.h"
+#import "FIROptionsInternal.h"
```

Comment:
Were the old repo-relative paths out of date then?

Reply:
It's now a public header for module construction - so cannot use paths.